homeassistant/auth_providers/homeassistant.py

```diff
@@ -44,9 +47,12 @@ def __init__(self, hass):
         if data is None:
             data = {
                 'salt': auth.generate_secret(),
-                'users': []
+                'users': [],
             }
 
+        if 'users' not in data:
```

Comment:
When can this happen?

Reply:
Data file was corrupted? Cannot recall why I added this statement, can be removed if you feel redundant.

Reply:
It's redundant. If storage doesn't exist, we get `None` back and set the default above. If we save, the default was the foundation.